lib/active_support/cache/memcached_store.rb

```diff
@@ -80,6 +80,12 @@ def initialize(*addresses, **options)
         extend Strategy::LocalCache
       end
 
+      def append(key, value, options = nil)
```

Comment:
Looks like `name` is elsewhere used to refer to the non-namespaced key

```suggestion
      def append(name, value, options = nil)
```